test/cpp/api/dataloader.cpp

```diff
@@ -1603,6 +1603,7 @@ TEST(DataLoaderTest, StatefulDatasetWithCollate) {
   ASSERT_TRUE(batch->target[0].allclose(torch::zeros(kBatchSize - 1)));
 }
 
+/*
```

Comment:
There should be a comment saying why this is commented out, and an issue tracking when we can uncomment it again.

But IIUC, you only wanted to disable this on Windows, right? Then it should be macro'ed.

Reply:
Are there pre-existing macros for this, or did you just mean to surround with `#ifdef`?